### ISSUE TITLE
Addition of Workflow's

### ISSUE THREAD

**Issue by shreykoradia** [shreykoradia/talez-frontend#13]
TLDR: Add a Dialog Model on Create Workflow Click Also add the Name and Description as a compulsary field.

How to add a workflow, we will just open a pop up or dialog box for allowing the user to add name and description (max of 250 words will increase the length of description from backend to around 550 words). Hence the Client Side will have to desgin a dialog box which could open on a click of create workflow and then allow the user to add the Name, no workflows will be without the Description hence the description will act as a compulsary field also try to maintain it with a Formik State
